src/test/scala/org/zalando/nakadi/client/KlientBuilderSpec.scala

```diff
@@ -8,21 +8,33 @@ import org.scalatest.{WordSpec, Matchers}
 class KlientBuilderSpec extends WordSpec with Matchers {
 
   "A Klient builder" must {
-    "must build a Klient instance, if everything is set properly" in {
+
```

Comment:
The "must" need not be repeated in the individual test messages. Note: it is really important what test messages say, because they may be read by not only programmers (this is one of the aspects we're trying to improve in Helsinki - anyone should be able to understand, what was tested, what failed etc.)